src/platform/EFR32/Logging.cpp

```diff
@@ -59,19 +59,22 @@
 #define LOG_DETAIL "<detail> "
 #define LOG_LWIP "<lwip  > "
 #define LOG_EFR32 "<efr32 > "
+static constexpr uint8_t kMaxCategoryStrLen = 9;
```

Comment:
Please document that this needs to be long enough to handle any of the LOG_* things (not including their null-terminator)?

Or better yet would be to compute it as https://github.com/project-chip/connectedhomeip/issues/4732#issuecomment-775393921 suggests, so it's obvious what's going on.  I just put up https://github.com/project-chip/connectedhomeip/pull/5124 to implement such a thing.

Reply:
I will update this seperatly with what you did in #5124.